src/alfasim_score/converter/alfacase/base_operation.py

```diff
@@ -0,0 +1,67 @@
+from typing import List
+
+import attr
+from alfasim_sdk import MassInflowSplitType
+from alfasim_sdk import MassSourceNodePropertiesDescription
+from alfasim_sdk import MassSourceType
+from alfasim_sdk import MultiInputType
+from alfasim_sdk import NodeDescription
+from alfasim_sdk import PressureNodePropertiesDescription
+
+from alfasim_score.constants import GAS_LIFT_MASS_NODE_NAME
+from alfasim_score.constants import NULL_VOLUMETRIC_FLOW_RATE
+from alfasim_score.constants import WELLBORE_BOTTOM_NODE_NAME
+from alfasim_score.constants import WELLBORE_TOP_NODE_NAME
+from alfasim_score.converter.alfacase.convert_alfacase import ScoreAlfacaseConverter
+from alfasim_score.converter.alfacase.score_input_reader import ScoreInputReader
+
+
+class BaseOperationBuilder(ScoreAlfacaseConverter):
+    def __init__(self, score_reader: ScoreInputReader):
+        super().__init__(score_reader)
+
+    def build_nodes(self) -> List[NodeDescription]:
+        operation_data = self.score_input.read_operation_data()
+        default_nodes = {node.name: node for node in super().build_nodes()}
+        configured_nodes = [
+            attr.evolve(
+                default_nodes.pop(WELLBORE_TOP_NODE_NAME),
+                mass_source_properties=MassSourceNodePropertiesDescription(
+                    temperature_input_type=MultiInputType.Constant,
+                    source_type=MassSourceType.AllVolumetricFlowRates,
+                    volumetric_flow_rates_std={
+                        "gas": -1.0
```

Comment:
Use the alfasim-sdk constant for phase.

Reply:
I don't get it. constant for phase? Do you mean I should ensure `volumetric_flow_rates_std_input_type=MultiInputType.Constant` ?

Reply:
Changed names of phases to use the constant values from alfasim-sdk `GAS_PHASE`, `OIL_PHASE`, `WATER_PHASE`